vnext/Microsoft.ReactNative.Managed/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,18 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Microsoft.ReactNative.Managed")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
```

Comment:
>"" [](start = 27, length = 2)

Microsoft?

Reply:
We have a lot of these that don't mention Microsoft, I used existing templates.
I'll clean all these up with a separate PR if that is okay.

---
In reply to: [420385179](https://github.com/microsoft/react-native-windows/pull/4793#discussion_r420385179) [](ancestors = 420385179)

Reply:
Sounds good!